### PR TITLE
bump mate de to 1.14

### PR DESCRIPTION
**Pull request by geki-yaba**
Well, summary says it.

Added mate-extra/mate-indicator-applet (version 1.14.0) and x11-themes/mate-themes (version 3.18.1).

### CODE REVIEW

**Comment by oz123** [comment]
Hi, 

Thanks for doing all that job! I really appreciate this ! Can you split this big commit to more smaller commits? 
It is very hard reviewing such a big commit. Especially, since I am extremely busy these days. Thanks, 
Oz

**Comment by geki-yaba** [comment]
Well, no idea how to change that. But there are classic tools(diff -u) that do the job just fine. Will do soonish.

**Comment by geki-yaba** [comment]
http://geki.selfhost.eu/hacks/mate-1.14.diff

**Comment by oz123** [comment]
Hi, I started reviewing your ebuilds. I noticed the copyright year is still 2015. Can you update those?

**Comment by oz123** [comment]
@geki-yaba at the current state, I can't merge your branch. I will however use your work in parts.

**Comment by geki-yaba** [comment]
As one can see in the diff, a handful lines changed and mate-extra/mate-indicator-applet added. most is 'cp a b'. far too simple.

**Comment by oz123** [comment]
Yes, but these ebuilds cause some havock together with mate 1.12. 
I just made a lot of changes in the ebuilds of mate 1.12. See these changes. 
Also just copying the files isn't always enough, in some cases you need to bump versions of Depends and rdepends inside the ebuilds.

Also note, I added one more comment on your commits, so that you'll see how to contribute. I very much welcome more people helping me. I know it's not trivial, but I'll do my best to guide as how to build ebuilds which conform to gentoo standards (so eventually they will be available in the main repository).

File: mate-base/mate-session-manager/mate-session-manager-1.14.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+GCONF_DEBUG="yes"
+
+inherit autotools gnome2 versionator
```
Comment (oz123) [inline mate-base/mate-session-manager/mate-session-manager-1.14.0.ebuild:9]: you can't inherit autotools, it does not exist. 
Please use repoman to validate your ebuilds.

**Pull request closed without merging**